### ISSUE TITLE
Employer_Role_IG_firm/model.xml is not valid XML

### ISSUE THREAD

**Issue by Allzi** [ETACE/eurace_unibi#11]
File eurace_unibi/Labour/Employer_Role_IG_firm/model.xml is not valid xml.

- opening tag for 'agents' is missing
- lines 599-604 are rubbish
- 'messages' element present in other model files is replaced with 'message' element

**Comment by Allzi** [comment]
Also closing tag for 'agents' is missing in Statistical_Office_IGFirm_real.xml
All other xml-files parse just fine.

**Comment by svdhoog** [comment]
Thanks for the checks! The Employer_Role_IG_firm is deprecated code and switched off in the main xml file, eurace_model.xml. It has actually been abandoned code for a few years, at least since 2014.
In all applications of the Eurace@Unibi model, we have no employees in the investment goods firm (IGFirm).

Note that in `eurace_model.xml` we have a dummy model active:

```
  <model>
   <file>Labour/Employer_Role_IG_Firm_dummy/model.xml</file>
   <enabled>true</enabled>
  </model>
```

**Comment by Allzi** [comment]
Fair enough. I was a bit surprised about the broken xml when parsing that to more readable format.
RIP that part of the code.
